web/src/stream_create.ts

```diff
@@ -508,15 +508,6 @@ export function show_new_stream_modal(): void {
             );
         }
     }
-    const $add_subscribers_container = $(
-        "#stream_creation_form .subscriber_list_settings",
-    ).expectOne();
-
-    stream_ui_updates.enable_or_disable_add_subscribers_elements(
-        $add_subscribers_container,
-        settings_data.can_subscribe_others_to_all_streams(),
-        true,
-    );
```

Comment:
Might be worth leaving a tombstone comment here of the form:

> We do not call stream_ui_updates.enable_or_disable_add_subscribers_elements here because ...

though I think it could be more readable to just call this, but pass `true` for whether we can add subscribers, with a comment above that line explaining why.